FWCore/ParameterSet/scripts/edmConfigDump

```diff
@@ -19,7 +20,10 @@ parser.add_argument("--pruneVerbose", action="store_true",
 
 options = parser.parse_args()
 handle = open(options.filename, 'r')
-cfo = imp.load_source("pycfg", options.filename, handle)
+loader=SourceFileLoader("pycfg", options.filename)
+loaded = types.ModuleType("pycfg",loader.name)
+loader.exec_module(loaded)
+#cfo = imp.load_source("pycfg", options.filename, handle)
```

Comment:
Is the commented out line still useful? It seems to me that `handle` becomes now unnecessary.